src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -596,24 +596,50 @@ def computer_test(user, print_traceback, computer):
 
 
 @verdi_computer.command('delete')
+@options.DRY_RUN()
 @arguments.COMPUTER()
 @with_dbenv()
-def computer_delete(computer):
+def computer_delete(computer, dry_run):
     """Delete a computer.
 
-    Note that it is not possible to delete the computer if there are calculations that are using it.
+    Note prompt can delete the computer even if running calculations are using it.
```

Comment:
What do you mean with "prompt can delete"? Do you mean this perhaps?
```suggestion
    Note: a computer can be deleted even if calculations are currently running on it. The calculation nodes will be
    deleted with the computer if confirmed but the calculations will not be automatically cancelled with the scheduler
```